inst/rmarkdown/templates/glossa/skeleton/sample.bib

```diff
@@ -0,0 +1,225 @@
+%% This BibTeX bibliography file was created using BibDesk.
```

Comment:
I did not check: Are you using all the references in this demo file ? 

If not maybe we can just keep what is necessary.

Reply:
There were a few extra refs in the bib file. I have removed them.